crates/tauri/src/event/mod.rs

```diff
@@ -123,21 +178,18 @@ impl FromStr for EventTarget {
 /// Serialized emit arguments.
 #[derive(Clone)]
 pub struct EmitArgs {
-  /// Raw event name.
-  pub event_name: String,
   /// Serialized event name.
   pub event: String,
   /// Serialized payload.
   pub payload: String,
 }
 
 impl EmitArgs {
-  pub fn new<S: Serialize>(event: &str, payload: S) -> crate::Result<Self> {
+  pub fn new<S: Serialize>(event: EventName<&str>, payload: S) -> crate::Result<Self> {
     #[cfg(feature = "tracing")]
     let _span = tracing::debug_span!("window::emit::serialize").entered();
     Ok(EmitArgs {
-      event_name: event.into(),
-      event: serde_json::to_string(event)?,
+      event: serde_json::to_string(event.as_str())?,
```

Comment:
Let's remove this serialization, it is not needed
```suggestion
      event,
```

don't forget to also update `emit_js_script` function down below to add wrapping quotes in the JS script.